src/main/java/com/esri/samples/featurelayers/feature_layer_rendering_mode_map/FeatureLayerRenderingModeMapSample.java

```diff
@@ -0,0 +1,164 @@
+      mapTop.getLoadSettings().setPreferredPolylineFeatureRenderingMode(FeatureLayer.RenderingMode.STATIC);
+      mapTop.getLoadSettings().setPreferredPolygonFeatureRenderingMode(FeatureLayer.RenderingMode.STATIC);
+
+      // create a map (bottom) and set it to render all features in dynamic rendering mode
+      ArcGISMap mapBottom = new ArcGISMap();
+      mapBottom.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+      mapBottom.getLoadSettings().setPreferredPolylineFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+      mapBottom.getLoadSettings().setPreferredPolygonFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+
+      // creating top map view
+      mapViewTop = new MapView();
+      mapViewTop.setPrefHeight(stage.getHeight() / 2);
+      mapViewTop.setMap(mapTop);
+      borderPane.setTop(mapViewTop);
+      // creating bottom map view
+      mapViewBottom = new MapView();
+      mapViewTop.setPrefHeight(stage.getHeight() / 2);
+      mapViewBottom.setMap(mapBottom);
+      borderPane.setCenter(mapViewBottom);
+
+      // create service feature table using a point, polyline, and polygon service
+      ServiceFeatureTable poinServiceFeatureTable = new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Energy/Geology/FeatureServer/0");
+      ServiceFeatureTable polylineServiceFeatureTable = new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Energy/Geology/FeatureServer/8");
+      ServiceFeatureTable polygonServiceFeatureTable = new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Energy/Geology/FeatureServer/9");
+
+      // create feature layer from service feature tables
+      FeatureLayer pointFeatureLayer = new FeatureLayer(poinServiceFeatureTable);
+      FeatureLayer polylineFeatureLayer = new FeatureLayer(polylineServiceFeatureTable);
+      FeatureLayer polygonFeatureLayer = new FeatureLayer(polygonServiceFeatureTable);
+
+      // add each layer to top and bottom scene
+      mapTop.getOperationalLayers().add(pointFeatureLayer);
+      mapTop.getOperationalLayers().add(polylineFeatureLayer);
+      mapTop.getOperationalLayers().add(polygonFeatureLayer);
+      mapBottom.getOperationalLayers().add(pointFeatureLayer.copy());
+      mapBottom.getOperationalLayers().add(polylineFeatureLayer.copy());
+      mapBottom.getOperationalLayers().add(polygonFeatureLayer.copy());
+
+      // viewpoint locations for map view to zoom in and out to
+      Viewpoint zoomOutPoint = new Viewpoint(new Point(-118.37, 34.46, SpatialReferences.getWgs84()), 650000, 0);
+      Viewpoint zoomInPoint = new Viewpoint(new Point(-118.45, 34.395, SpatialReferences.getWgs84()), 50000, 90);
+      mapViewTop.setViewpoint(zoomOutPoint);
+      mapViewBottom.setViewpoint(zoomOutPoint);
+
+      Button zoomButton = new Button("Start Zoom");
```

Comment:
Instead of a button and all of the done listeners, you could use a looping animation. I think it would be cool to show this in a sample:
```java
//loop an animation into and out from the zoom in point (5 seconds each) with a 2 second gap between zooming
Timeline timeline = new Timeline();
timeline.setCycleCount(Animation.INDEFINITE);
timeline.getKeyFrames().add(new KeyFrame(Duration.seconds(7), event -> zoomTo(zoomInPoint, 5)));
timeline.getKeyFrames().add(new KeyFrame(Duration.seconds(14), event -> zoomTo(zoomOutPoint, 5)));
timeline.play();
```